src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -41,12 +46,35 @@ def test_allow_auto_batching_property():
                     reason=f"Cannot run test on device {os.environ.get('TEST_DEVICE')}, Plugin specific test")
 def test_single_property_setting():
     core = Core()
+    device_name = core.get_property("CPU", "FULL_DEVICE_NAME")
+    if "Intel" not in device_name:
+        pytest.skip("Properties are Intel(R) CPU specific")
```

Comment:
I suppose this property also works for ARM CPU, GPU, MYRIAD, HDDL

Reply:
This test file was updated separately here: https://github.com/openvinotoolkit/openvino/pull/14684